spec/forms/citizens/percentage_home_form_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe Citizens::PercentageHomeForm, type: :form do
+  let(:percentage_home) { Faker::Number.decimal(2).to_f }
+  let(:application) { create :legal_aid_application }
```

Comment:
Why do we need to weed them out? What's your argument? I personally prefer `application`. It's shorter, it works, and I think it's quite obvious it is the legal aid application rather than the Rails application

Reply:
OK - perhaps I'm being perdantic, but the object is `LegalAidApplication` and not `Application`. I may be the only one to have a problem with this.